service/pixelated/resources/root_resource.py

```diff
@@ -40,10 +40,10 @@
 
 
 class RootResource(BaseResource):
-    def __init__(self, services_factory):
+    def __init__(self, services_factory, static_folder=None):
```

Comment:
I don't understand the question precisely.
They are "custom" in the sense that I need to control where they are (ie, inside a virtualenv or on the top-level folder from a froze bundle).
I find this solution more straightforward that the _get_static_folder() below, which seems a bit hacky.